packetbeat/docs/configuration.asciidoc

```diff
@@ -199,6 +200,34 @@ you use this setting, it's your responsibility to keep the BPF filters in sync w
 ports defined in the `protocols` section.
 
 
+[[configuration-flows]]
+=== Flows
+
+The `flows` section contains configuration options for bidirectional network
+flows. If section is missing from configuration file, network flows are
+disabled.
```

Comment:
If flows are enabled the BPF filter is set to "" (-> pass all packets), unless user configures a BPF filter in the sniffer section.